### ISSUE TITLE
Crash when using BottomNavigationView from latest material library

### ISSUE THREAD

**Issue by eriksquare** [phrase/phrase-android#29]
I'm using the latest material components version (1.4.0-rc01) and when accessing a screen which hosts the BottomNavigationView component in my app i receive this crash:
```
java.lang.NoSuchFieldError: No static field BottomNavigationView_menu of type I in class Lcom/google/android/material/R$styleable; or its superclasses (declaration of 'com.google.android.material.R$styleable' appears in /data/app/com.example.debug-Wmm5KDB8jGv2oazN9rPPWQ==/base.apk!classes17.dex)
        at com.phrase.android.sdk.inject.ViewTransformer.a(SourceFile:14)
        at com.phrase.android.sdk.inject.ViewTransformer.transformView(SourceFile:213)
        at androidx.appcompat.app.PhraseAppCompatDelegate.createView(SourceFile:10)
        at androidx.appcompat.app.PhraseAppCompatDelegate$factory2$1.onCreateView(SourceFile:1)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:189)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:772)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at androidx.fragment.app.Fragment.onCreateView(Fragment.java:1933)
```

I also see this issue on previous 1.3.0 version of the material library. 
I'm on the latest version (3.0.5) of the phrase sdk, and can confirm that by removing it the issue is solved.

**Comment by theSoenke** [comment]
Hi @eriksquare, we are currently investigating this case with a `BottomNavigationView` and I'll provide an update as soon as I have more information

**Comment by theSoenke** [comment]
We investigated what is causing this issue and it turns out that `BottomNavigationView` is a subclass of `NavigationBarView` since `com.google.android.material:material:1.4.0-rc01`. 

Are you already relying on the release candidate of the material library? If not I would suggest waiting for a stable 1.4.0 release and then shortly after we will add support for 1.4.0. If you already rely on the release candidate of the material library we could create a release candidate of our SDK, which includes a fix for this issue.

**Comment by eriksquare** [comment]
Yeah, no issue from me with that, since at the moment we are using this component only for internal development.

**Comment by eriksquare** [comment]
material library 1.4.0 has been released as stable

**Comment by theSoenke** [comment]
@eriksquare Thanks for letting us know. This should be solved in version 3.0.6 of the SDK
